### PR TITLE
totaltime calculation working

### PR DESCRIPTION
**Pull request by piknik1979**

### CODE REVIEW

File: src/components/Game.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
-import { useState, useEffect } from "react";
```
Comment (yuridevweb) [inline src/components/Game.js:1]: Can we be more consistent and do import in the same way across the app?

File: src/components/Timer.js [hunk 2]
```diff
@@ -1,56 +1,34 @@
-import * as React from "react";
```
Comment (yuridevweb) [inline src/components/Timer.js:1]: and this

**Review by yuridevweb — COMMENTED** [review]
Looks great, just these little remarks

File: src/components/Game.js (hunk 1 above)
Comment (ZanClifton) [reply src/components/Game.js:1]: Agreed, we are taking two different approaches. We should discuss.

**Comment by ZanClifton** [comment]
Excellent work! Well done! Let's merge!

**Pull request merged**